ragstack/colbert/colbert_embedding.py

```diff
@@ -129,19 +148,30 @@ def encode_queries(
         query: Union[str, List[str]],
         full_length_search: bool = False,
         query_maxlen: int = 32,
+        enable_dynamic_query_maxlen: bool = True,
```

Comment:
is there anyway to specify this with one parameter? like maybe if `query_maxlen` isn't set, then we use dynamic?

Reply:
fxied